content/en/community/mission.md

```diff
@@ -1,13 +1,7 @@
 ---
-title: Mission
+title: Mission, vision, and values
+aliases: [/mission, /vision, /values]
```

Comment:
If ever there is a decision to move ahead with this page move, I'd vote against introducing two new aliases for pages that did not exist before, namely `/vision`, and `/values`.

Reply:
fixed